src/com/flickity.js

```diff
@@ -88,87 +88,106 @@
+					t.focus();
 				}
-				
-				if (!!flickOptions.pauseOnInit) {
-					slider.flickity('pausePlayer');
+			});
+			//set slide aria-label on corresponding page dot
+			slider.find('.flickity-page-dots li').each(function (i) {
+				var t = $(this);
+				var ariaLabel = slider.find(o.cellSelector).eq(i).attr('aria-label');
+				if (!!ariaLabel) {
+					t.attr('aria-label', ariaLabel);
 				}
+			});
+			// Limit focus to active slide's content only
+			slider.on('blur', function () {
+				slider.data('acc-tabindex', slider.attr('tabindex'));
+				slider.removeAttr('tabindex');
+			})
+			.on('settle.flickity', function () {
+				var curSlide = getCurrentSlide();
+				slider.find(o.cellSelector).removeAttr('tabindex');
+				slider.find('button, a').attr('tabindex', '-1');
+				curSlide.add(curSlide.find('button, a')).attr('tabindex', '0');
 				
-				// restart autoplay slider after user action for desktop devices
+				// restart autoplay slider after user action for touch devices
 				if (!!flickOptions.autoPlay &&
 					!!flickOptions.restartAutoplayOnMouseleave &&
-					!$.mobile) {
-					slider.on('mouseleave', sliderPlay);
+					$.mobile) {
+					sliderPlay();
+				}
+			});
+			if (!!flickOptions.pageDots) {
+				slider.on('settle.flickity', setActiveSlideSeen);
+			}
+			slider.find('img[data-src-format]').jitImage();
+			if (!!flickOptions.pauseOnFocus) {
+				// Make sure it is focusable
+				var tab = slider.attr('tabindex');
+				if (!tab) {
+					slider.attr('tabindex', '0');
 				}
+				slider.on('focus', function (e) {
+					slider.flickity('pausePlayer');
+				}).on('focus', '*', function (e) {
+					slider.flickity('pausePlayer');
+				});
+			}
+			
+			if (!!flickOptions.pauseOnInit) {
+				slider.flickity('pausePlayer');
+			}
+			
+			// restart autoplay slider after user action for desktop devices
+			if (!!flickOptions.autoPlay &&
+				!!flickOptions.restartAutoplayOnMouseleave &&
+				!$.mobile) {
+				slider.on('mouseleave', sliderPlay);
+			}
+			
+			// Check for visibility
+			if (!!window.IntersectionObserver) {
+				intersectionObserver = new window.IntersectionObserver(function (entries) {
+					var e = entries[0];
+					var ap = !!flickOptions.autoPlay || !!flickOptions.autoPlayWhenVisible;
+					if (!e) {
+						return;
+					}
+					if (e.isIntersecting && ap) {
+						sliderPlay();
+					} else if (!e.isIntersecting) {
+						sliderPause();
+					}
+				});
+				intersectionObserver.observe(slider.get(0));
+			}
+		};
+
+		var init = function (item) {
+			slider = item;
+
+			if (slider.find(o.cellSelector).length > 1) {
+				initWithMultipleCells(slider);
 				
 				App.callback(o.inited);
 			} else if (slider.find(o.cellSelector).length === 1) {
 				slider.addClass(o.abortedClass);
 				slider.find(o.cellSelector).addClass(o.selectedClass);
 				App.callback(o.aborted);
 			} else {
-				App.log('No flickity found or multiple flickity in the scope');
+				App.log('No flickity found cell found in the scope');
```

Comment:
I think there is too many "found" in this sentence..

Reply:
@dominic-blain Good catch 👍 Fixed in a0830f1